src/tools/tidy/src/pal.rs

```diff
@@ -58,7 +58,7 @@ const EXCEPTION_PATHS: &[&str] = &[
     "src/rtstartup", // Not sure what to do about this. magic stuff for mingw
 
     // temporary exceptions
-    "src/libstd/lib.rs", // FIXME(#44217)
```

Comment:
I'd be curious to know if the original intention was that "src/libstd/lib.rs" would eventuallyi not be on this list, but #44217 was blocking that. In which case, what is the intention now...?

Reply:
right now tidy complains about sgx specific `cfg_attr` in [here](https://github.com/rust-lang/rust/blob/01c6ea2f37326674bf6ca64de55bf5fd90e45015/src/libstd/lib.rs#L316).